resources/assets/js/src/components/pages/Login.tsx

```diff
@@ -95,8 +101,34 @@ class Login extends React.Component<any> {
 
     private submitHandler(e: any): void {
         e.preventDefault();
+
+        if (!this.state.formValues.email || !this.state.formValues.password) {
```

Comment:
I would prefer to use react-validation package https://www.npmjs.com/package/react-validation

Reply:
That would cause problems with react-bootstrap, because it's using hocs for form elements

Reply:
Please try to look for any similar library for react-bootstrap

Reply:
That's not problem with react-bootstrap, it will cause same problem for any library that wrap css classes to simplify jsx structure

Reply:
I tried all packages (react-validate, react-validation, react-bootstrap-validation) and all of them causes errors or they are deprecated.